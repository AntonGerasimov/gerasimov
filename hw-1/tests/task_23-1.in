1 101 1.1
