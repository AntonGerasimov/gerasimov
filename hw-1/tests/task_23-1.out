1
1.100000
e
